target/linux/ath79/dts/qca9563_tplink_cpe710-v1.dts

```diff
@@ -0,0 +1,145 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+        model = "TP-Link CPE710 v1";
+        compatible = "tplink,cpe710-v1", "qca,qca9563";
+
+
+	aliases {
+		label-mac-device = &eth0;
+                led-boot = &led_lan;
+                led-failsafe = &led_lan;
+                led-upgrade = &led_lan;
+
+	};
+
+        leds {
+                compatible = "gpio-leds";
+
+                led_lan: lan {
+                        label = "blue:lan";
+                        gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+                };
+
+                wlan5g {
+                        label = "blue:wlan5g";
+                        gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+                        linux,default-trigger = "phy0tpt";
+                };
+        };
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "Reset button";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&spi {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <25000000>;
```

Comment:
will higher frequencies work as well, e.g. `50000000`?

Reply:
I have kept it as 25000000 to be consistent with the other TP-LINK CPE Devices as well the the other devices based on the AP152 board

Reply:
actually, max frequency for the flash is not something "consistent" based on the board model, but rather flash model. The other CPE devices have 8 MB chips. Almost all 16 MB chips are capable of at least 40 MHz

(btw it's nice to have flash chip model in commit message, you can find it in kernel log if you don't want to open it up)
(also there is public images of the inside via the FCC ID)

Reply:
The TP-LINK EAP devices with a 16MB Flash also based on the AP152 Reference Board all have 25000000 as well.

Reply:
those EAP models would also benefit from max frequency being bumped up to 40 MHz

Reply:
What is the exact value I should try and is there any risk of it shortening the life of the flash?

Reply:
there is no risk to the life of the flash here, it either reads or it doesn't

replace the "25" with "40" for 40 MHz